src/plugin.ts

```diff
@@ -64,11 +86,25 @@ const plugin: FastifyPluginAsync = async (fastify) => {
       const task = taskManager.createUpdateItemValidationReviewTask(
         member,
         id,
-        data as Partial<ItemValidationReview>,
+        data as {status: string, reason: string},
       );
       return runner.runSingle(task, log);
     },
   );
+
+    // update item validation process
+    fastify.post<{ Params: { id: string } }>(
+      '/validations/process/:id/enabled',
+      { schema: itemValidationProcess },
+      async ({ member, params: { id }, body: data, log }) => {
+        const task = taskManager.createToggleEnabledForItemValidationProcessTask(
+          member,
+          id,
+          data as {enabled: boolean},
```

Comment:
I think it's possible to set the Body type (next to Params) instead of casting it. Same for above endpoints.